model_finder.py

```diff
@@ -0,0 +1,50 @@
+'''
+Script to help choose best model visually
+For 1 vh vv pair
+Predict mask for every model in models
+'''
+
+import os
+import subprocess
+from tkinter import *
+from tkinter.filedialog import askopenfilename
+from pathlib import Path
+
+
+def getFileName():
+    root = Tk()
+    filename = askopenfilename()
+    root.destroy()
+    return filename
+
+
+def main():
+    # get inputs
+    model_path = 'models/ai_test_5'
```

Comment:
We might not want to hard code in ai_test_5

Reply:
removing